geoutils/__init__.py

```diff
@@ -2,20 +2,18 @@
 GeoUtils is a python package of raster and vector tools.
 """
 
-from . import georaster
-from . import geovector
-from . import datasets
-from . import satimg
-
-from .georaster import Raster
-from .geovector import Vector
-from .satimg import SatelliteImage
+from geoutils import datasets, georaster, geovector, satimg  # noqa
```

Comment:
what does noqa mean?

Reply:
I don't know what it stands for, but it's for disabling `flake8` lints. In this case, it warns about unused imports (which is as it should in an `__init__.py` file)